docs/ADRs/README.md

```diff
@@ -0,0 +1,7 @@
+# Documentation of design decisions that have been made in Poliaris
+
+Polaris is an opinionated scafolding. Decisions have been made as to the technologies used therein, and these are documented here.
```

Comment:
isn't that scaffolding with 2 `f`
btw: TIL the word `therein` thanks @katie-roberts